### DISCUSSION TITLE
Certain package won't load on website + CLI

### DISCUSSION THREAD

**Discussion by JFulweber** [npm/feedback#1024] [Website]
Going to the NPM page for @vueup/vue-quill gives an error code for the page itself, and fails to install with npm install. Not sure if this is the packages fault or NPMs.

https://www.npmjs.com/package/@vueup/vue-quill

**Comment by z2oh** [comment]
Also seeing this for https://www.npmjs.com/package/@vscode/vsce and https://www.npmjs.com/package/@vscode/test-electron . I'm assuming it has to do with the @ in the package name. https://status.npmjs.org/ just updated saying there's a package install issue underway.

**Comment by JFulweber** [comment]
Looks like it's been taken care of
